src/styles/blocks/footer.scss

```diff
@@ -0,0 +1,6 @@
+.footer {
+  height: 400px;
+  background-image: url(../images/photos/image-footer.jpg);
```

Comment:
Consider adding a fallback background color before the background image, in case the image fails to load. For example: `background-color: $fallback-color;` 💡